src/main/integrations/gpu-context.ts

```diff
@@ -0,0 +1,76 @@
+import { defineIntegration, dropUndefinedKeys, Event } from '@sentry/core';
+import { app } from 'electron';
+
+interface Options {
+  infoLevel: 'basic' | 'complete';
```

Comment:
Could we add docs for this? We can link to https://www.electronjs.org/docs/latest/api/app#appgetgpuinfoinfotype